dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -293,6 +298,10 @@ const frontsBannerAdStyles = css`
 	max-width: ${breakpoints['wide']}px;
 	overflow: hidden;
 	padding-bottom: ${frontsBannerPaddingHeight}px;
+	${between.tablet.and.desktop} {
+		max-width: ${adSizes.leaderboard.width}px;
+		max-height: ${90 + labelHeight}px;
```

Comment:
```suggestion
		max-height: ${adSizes.leaderboard.height + labelHeight}px;
```

Reply:
I'd be tempted to make this max-width and max-height the default, like so:
```
	position: relative;
	min-height: ${frontsBannerMinHeight}px;
	/* No banner should be taller than 600px */
	max-height: ${adSizes.leaderboard.height + labelHeight}px;
	max-width: ${adSizes.leaderboard.width}px;
	overflow: hidden;
	padding-bottom: ${frontsBannerPaddingHeight}px;
	${from.desktop} {
		max-height: ${600 + labelHeight}px;
	    max-width: ${breakpoints['wide']}px;	
    }
```

Reply:
I see so it working up incrementally as the breakpoint increases. 👍

Reply:
@domlander refactored 👍